apps/ui/src/composables/useSpaces.ts

```diff
@@ -143,21 +144,28 @@ export function useSpaces() {
   }
 
   async function fetchMore(filter?: SpacesFilter) {
-    if (loading.value || !loaded.value) return;
+    if (loading.value || !loaded.value || loadingMore.value) return;
     loadingMore.value = true;
 
     await _fetchSpaces(false, filter);
 
     loadingMore.value = false;
   }
 
+  watch(protocol, async (newProtocol: ExplorePageProtocol) => {
+    protocol.value = newProtocol;
```

Comment:
Do we need it? Looks like we are writing `protocol` second time when it's written to protocol before.